src/static/js/modules/prepare-worksheets/inputs/button-grading-group.js

```diff
@@ -11,42 +11,39 @@ function ButtonGradingGroup(options) {
   // Note bind()'s lack of IE8 support.
   var _self = this;
 
-  var _grades = require('./input-graded-grades');
-
-  // The grade of this input.
-  var grade = _grades.UNSET;
-  var lastNode;
-
+  this.row = options.row;
```

Comment:
Good point! There's no need for that to be exposed externally, so I'll scope it locally.